doc/man1/openssl-fipsinstall.pod.in

```diff
@@ -163,6 +164,11 @@ fails as described above.
 
 Configure the module to not perform run-time security checks as described above.
 
+=item B<-self_test_onload>
+
+Configure the self tests KATS to always run when the module is loaded. If this
+option is not used then the self tests KAT's are only run once during installation.
+
```

Comment:
I'm not sure I understand the purpose of this added flag...  doesn't this have the same effect as simply not producing the fipsmodule configuration?

Reply:
No, it will just not output the install-mac and install-status. The rest will be the same.

Reply:
Without the configuration, there is no checksum so the module won't start?

Reply:
Aha, ok then

Reply:
Who needs run-KAT-each-time?  It's not FIPS 140-2 that is requiring this.

Reply:
No, FIPS 140-2 is not requiring this, however only if you can assure that you run KATs at least once on the deployment. It would be for example breaking FIPS requirements if you build at one machine, create the fipsmodule.cnf with the indicator that KATs were run (on the build machine) and then deploy it elsewhere. The KATs would not be run at the deployment machine at all and that breaks FIPS requirements.

Reply:
I understand it must be run "at install time"  Isn't putting that into the Security Policy enough?  What is the expected documentation around when to use this flag? How does it solve the problem of "you didn't do it at install time"? And is the solution worse (a seeming run-time flag) than the real issue (RTFM) :)

Reply:
This PR is not adding the runtime flag. It is already there. It just adds convenience option to the fipsinstall command.

Reply:
Thank you for the clarification. I missed the original addition of the runtime flag I guess.

Reply:
You need the run-time flag to be actually able to "run the KATS at least once" so it had to be always there.